### PR TITLE
Expose helix participant log config

### PR DESCRIPTION
**Pull request by sihanwang-pinterest**

### CODE REVIEW

File: rocksdb_admin/helix_client.cpp [hunk 1]
```diff
@@ -27,17 +27,18 @@
 #include "common/network_util.h"
 
 DECLARE_int32(port);
+DEFINE_string(helix_log_config, "", "Participant log config");
```
Comment (newpoo) [inline rocksdb_admin/helix_client.cpp]: essentially this can be any flags passed to JVM? If yes, shall we rename it?

Comment (sihanwang-pinterest) [reply rocksdb_admin/helix_client.cpp]: updated

File: rocksdb_admin/helix_client.cpp [hunk 2]
```diff
@@ -47,6 +50,7 @@ JNIEnv* createVM(const std::string& class_path) {
   options[1].optionString = "-verbose:jni";
   // use -Xrs to block JVM signal handling
   options[2].optionString = "-Xrs";
+  options[3].optionString = const_cast<char*>(FLAGS_helix_log_config_file_path.c_str());
```
Comment (newpoo) [inline rocksdb_admin/helix_client.cpp]: could you please verify that it will work fine even the gflag is set to empty string?

Comment (sihanwang-pinterest) [reply rocksdb_admin/helix_client.cpp]: Updated the pull request, and tested with empty string and arbitrary string, jvm will ignore the wrong config.

**Pull request merged**